src/main/java/Duke.java

```diff
@@ -1,10 +1,138 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
```

Comment:
Duke class and main method is very long, consider having smaller classes to handle different commands.